content/docs/02.tutorial/06.errors.md

```diff
@@ -5,11 +5,11 @@ title: Errors and Retries
 By default, a failure of any task will stop the execution and will mark it as failed.
 For more control over error handling, you can add `errors` tasks, `AllowFailure` tasks, or automatic retries.
 
-The error tasks allow you to execute one or more actions before terminating the flow, e.g. sending an e-mail or a Slack message to your team. The property is named `errors` because they are triggered upon errors within your flow.
+The `errors` tasks allow you to execute one or more actions before terminating the flow, e.g. sending an email or a Slack message to your team. The property is named `errors` because they are triggered upon errors within your flow.
```

Comment:
```suggestion
The `errors` property allows you to execute one or more actions before terminating the flow, e.g. sending an email or a Slack message to your team. The property is named `errors` because they are triggered upon errors within your flow.
```